types/standard/config/tests-travis.webpack.js

```diff
@@ -0,0 +1,3 @@
+var context = require.context('../../../../../../js', true, /-test\.jsx?$/);
```

Comment:
this imposes the position in node modules. I think you should use paths here, if possible. Same for tests.webpack.js